src/net/atlas/db.rs

```diff
@@ -462,31 +651,82 @@ impl AtlasDB {
         Ok(row)
     }
 
-    pub fn insert_uninstantiated_attachment_instance(
+    /// Queue a new attachment instance, status will be set to "queued",
+    /// and the is_available field set to false
+    pub fn queue_attachment_instance(
```

Comment:
Can you document which thread calls this function?  Same for `insert_initial_attachment_instance`.

Reply:
Sure, however, I'm somewhat hesitant to document _the caller_ from the called method. If atlas event processing is moved from the coordinator's purview to `db::blocks` or whatever, that move can occur without ever touching this function, which could easily lead to a stale rustdoc here.

Reply:
Added in https://github.com/stacks-network/stacks-blockchain/pull/3086/commits/cb9f8dca0f9697e7cfe04c1b2e776de4b8aec8ce